content/knowledge-base/migrate-to-provider-families.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Migrate to Provider Families
+---
+
+The Upbound official [provider families]({{<ref "upbound-marketplace/providers#provider-families">}}) organize provider resources into common, logical, groupings. 
+Provider families replace the original monolithic providers, which combined all provider resources into a single Provider package.
+
+Migrating from a monolithic provider to a family provider allows the family provider to take ownership of existing managed resources. 
+
+{{<hint "important" >}}
+Upbound only supports migration from Upbound's official monolithic providers. 
+
+Migration from Crossplane community providers to Upbound provider families isn't supported.
+{{< /hint >}}
+
+## Backup existing Crossplane resources
+
+Backup the existing Crossplane resources including managed resources, Compositions and Claims.
+
+In case of problems, Crossplane can restore the resources from the YAML files. 
+
+{{<hint "note" >}}
+This may take over 10 minutes depending on the number of managed resources in the cluster. 
+{{< /hint >}}
+
+```bash 
+kubectl get managed -o yaml > backup-mrs.yaml
+kubectl get composite -o yaml > backup-composites.yaml
+kubectl get claim --all-namespaces -o yaml > backup-claims.yaml
+```
+
+## Update the resource deletion policy
+
+Update the deletion policy on all managed resources to `Orphan` to prevent
+Crossplane from deleting the resources. 
+
+Find any resources that are already set to `Orphan`.
+
+```shell
+kubectl get $(kubectl get managed -o name) -o json | jq '.items[] | select(.spec.deletionPolicy=="Orphan") | "\(.spec.forProvider.tags."crossplane-kind")/\(.spec.forProvider.tags."crossplane-name")"'
+```
+
+Save the names of any returned resources. 
+
+Next, set the deletion policy of all managed resources to `Orphan`.
+
+```shell
+kubectl patch $(kubectl get managed -o name) -p '{"spec":{"deletionPolicy":"Orphan"}}' --type=merge
+```
+
+{{<hint "note" >}}
+These commands may take over 10 minutes depending on the number of managed resources in the cluster. 
+{{< /hint >}}
+
+## Create a list of family provider services
+
+Upbound maintains a migration script that maps all existing managed resources to their family provider packages.
+This script generates a list of new family providers to install. 
+
+1. Download the script
+```bash
+wget https://raw.githubusercontent.com/upbound/upjet/main/docs/sp-migration/generate-manifests.sh
```

Comment:
Could you please change this link with the following one? (As described, we are moving the docs from upjet to the new migration repo) https://github.com/upbound/extensions-migration/blob/main/docs/generate-manifests.sh

Reply:
updated, thanks!